modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -8828,23 +8828,23 @@ public static void removeJavaNoOpLogger(Collection<Handler> rmvHnds) {
     /**
      * Attaches node ID to log file name.
      *
-     * @param nodeId Node ID.
+     * @param postfix Postfix.
      * @param fileName File name.
      * @return File name with node ID.
      */
     @SuppressWarnings("IfMayBeConditional")
-    public static String nodeIdLogFileName(UUID nodeId, String fileName) {
-        assert nodeId != null;
+    public static String nodeIdLogFileName(String postfix, String fileName) {
```

Comment:
I suggest renaming the method and the javadoc as well.

Reply:
Fixed.